README.md

```diff
@@ -10,6 +10,10 @@
 > need to pin rspec itself to `~> 3.1.0`, as later rspec versions do not work
 > on old rubies anymore.
 
+> The spec tests for this project are configured to run against the versions of Puppet included with
+> [currently supported versions of Puppet Enterprise](https://puppet.com/misc/puppet-enterprise-lifecycle).
+> To work with older Puppet versions (e.g. Puppet 3.8.x), revert to the [v.2.5.0](https://github.com/rodjek/rspec-puppet/tree/v2.5.0) tag of this project.
```

Comment:
Nope. Please delete.

Thinking back, I think we've talked about this before understanding the complete matrix.